pickATrip_django_apps/admin.py

```diff
@@ -1,3 +1,32 @@
-# from django.contrib import admin
+from django.apps import apps
+from django.contrib import admin
+from commenting_system.models import Comment
 
-# Register your models here.
+
+# Customized model registration
+@admin.register(Comment)
+class CommentAdmin(admin.ModelAdmin):
+    list_display = ('user', 'body', 'created_on', 'tag', 'post', 'active')
+    list_filter = ('active', 'created_on')
+    search_fields = ('username', 'body')
+    actions = ['approve_comments']
+
+    def approve_comments(self, request, queryset):
+        queryset.update(active=True)
+
+
+# for viewing models registration fields
+class ListAdminMixin(object):
+    def __init__(self, general_model, admin_site):
+        self.list_display = [field.name for field in general_model._meta.get_fields()]
+        super(ListAdminMixin, self).__init__(model, admin_site)
+
+
+# Register all other models automatically - should stay last in file.
+models = apps.get_models()
+for model in models:
+    admin_class = type('AdminClass', (ListAdminMixin, admin.ModelAdmin), {})
+    try:
+        admin.site.register(model, admin_class)
+    except admin.sites.AlreadyRegistered:
+        pass
```

Comment:
what is this for?

Reply:
for automating the model registration in the admin for models that will be added later, in case they weren't registered manually as I did with more customization for the comment model, It only provides a way to interact with the models and viewing each table by its fields in the admin interface

Reply:
There is a way to do that, not needed to be admin
Please refer Task#5 in first challenge

Reply:
Sorry, can you elaborate here? it is not clear to me

Reply:
OK, reading this https://docs.djangoproject.com/en/3.1/ref/contrib/admin/
I think it should be removed from this PR, and proposed on different PR,
Can you open an Issue? such as, create  automatic admin interface w/ PseudoCode?
Please implement one thing in single PR

Reply:
@maayanhd @guy9050, any updates here? 
I choose not die on that hill :+1: